anime_downloader/sites/animeonline360.py

```diff
@@ -0,0 +1,55 @@
+import json, requests
+
+from anime_downloader.sites.anime import Anime, AnimeEpisode, SearchResult
+from anime_downloader.sites import helpers
+
+class AnimeOnline(Anime, sitename = 'animeonline360'):
+
+    sitename = 'animeonline360'
+    url = f'https://{sitename}.me'

```

Comment:
probably not needed if you're not using it

Reply:
ok